src/tools/dashboard/components/Row.tsx

```diff
@@ -258,6 +265,94 @@ const NiceStatus = ({ job, jobLink, jobUrl }: NiceStatusProps) => {
   }
 };
 
+interface JobSpecificParametersProps {
+  job: Job;
+}
+
+const taxonsWithEllipsisReveal = (taxIDs: SelectedTaxon[]) => {
+  const visibleID = taxIDs[0];
+  const hiddenIDs = taxIDs.slice(1);
+
+  return (
+    <span>
+      Selected taxonomy: {visibleID.label}
+      {hiddenIDs.length ? (
+        <EllipsisReveal>
+          {', '}
+          {hiddenIDs.map((taxon) => taxon.label).join(', ')}
+        </EllipsisReveal>
+      ) : null}
+    </span>
+  );
+};
+
+const JobSpecificParamaters = ({ job }: JobSpecificParametersProps) => {
+  const { data: idMappingFields } = useDataApi<IDMappingFormConfig>(
+    apiUrls.idMappingFields
+  );
+
+  const idMappingDBToDisplayName: { [k: string]: string } | undefined =
+    useMemo(() => {
+      const dbNameToDbInfo = idMappingFields
+        ? Object.fromEntries(
+            idMappingFields.groups.flatMap(({ items }) =>
+              items.map((item) => [item.name, item.displayName])
+            )
+          )
+        : {};
+      return dbNameToDbInfo;
+    }, [idMappingFields]);
+
+  if (job.status === Status.FINISHED) {
```

Comment:
Any specific reason to only display this when finished? I think it might be useful also when running, especially to understand why one might take long to finish

Reply:
Yeah, I think it might be useful for any job in any status